hathor/cli/run_node.py

```diff
@@ -376,6 +377,8 @@ def __init__(self, *, argv=None):
             os.environ['HATHOR_CONFIG_YAML'] = self._args.config_yaml
         elif self._args.testnet:
             os.environ['HATHOR_CONFIG_YAML'] = TESTNET_SETTINGS_FILEPATH
+        elif args.nano_testnet:
```

Comment:
Shouldn't it be `self._args.nano_testnet`?

Reply:
Fixed! Thanks!